internal/weldr/api.go

```diff
@@ -10,10 +10,12 @@ import (
 	"github.com/julienschmidt/httprouter"
 
 	"osbuild-composer/internal/rpmmd"
+	"osbuild-composer/queue"
```

Comment:
Libraries should be in either the `pkg` or `internal` subdirectory. For now, let's put everything in `internal`, and only if we want something to be public we put it in `pkg`.